runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
> I thought the point was start does not have to be called.

Yes.

> … thus we only have this MUST when start is called.

That's what I was going for with an [optional `process.args`][1] which still contains [a MUST for an initial argument][2].  That requirement (if `process.args` is set, it MUST contain that leading entry) applies to all configs, not just when `start` is called.

[1]: https://github.com/opencontainers/runtime-spec/pull/620/commits/35ee63bd00cee7da9401b9854620112c4d83d8e3#diff-c9c91c29b41257aea3a3403cc606ad99R131
[2]: https://github.com/opencontainers/runtime-spec/pull/620/commits/35ee63bd00cee7da9401b9854620112c4d83d8e3#diff-c9c91c29b41257aea3a3403cc606ad99R132